### PR TITLE
[#162144995] create login and sign up page

### PR DESCRIPTION
**Pull request by NabeelahY**
- Users are able to create an account
-Users are able to log into their account

### CODE REVIEW

File: UI/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<head>
+	<title>Welcome to iReporter</title>
+	<link rel="stylesheet" type="text/css" href="style/style.css">
+</head>
+<body>
+	<center>
+			<div class="land">
+				<h1>iReporters</h1>
+				<p>With the high rise of corruption and deploration of our country's infrastructures, it is time for the citizens to take charge of the situation. iReporter is a platform where citizens can share accounts of corruption witnessed or things that need urgent goverment intervention to the notice of appropriate authorities and the
+				general public.</p>
```
Comment (fuzzybrows) [inline UI/index.html]: the indenting here is too much and inconsistent. What Editor are you using?

File: UI/index.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+				<button type="submit"><a href="login.html">Login / Sign Up</a></button>
+			</div>
+	</center>
+
+</body>
+</html>
```
Comment (fuzzybrows) [inline UI/index.html]: also there should be an EOF line for every file.

File: UI/index.html (hunk 1 above)
Comment (NabeelahY) [reply UI/index.html]: I'm using Sublime

Comment (fuzzybrows) [reply UI/index.html]: This is still yet to change

**Pull request merged**